modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/http/ESHttpClient.java

```diff
@@ -106,7 +219,7 @@ public String getClusterName() {
 		try {
 			JestResult result = client.execute(nodesinfo);
 			clusterName = result.getValue("cluster_name").toString();
-		} catch (IOException e) {
+		} catch (IOException | NullPointerException e) {
```

Comment:
That was a dirty trick when the cluster name was not available through the API Call.
For instance, when syslog_ng has no permissions to do so:

```
{"error":{"root_cause":[{"type":"security_exception","reason":"no permissions for cluster:monitor/nodes/info"}],"type":"security_exception","r
eason":"no permissions for cluster:monitor/nodes/info"},"status":403}";
```